framework/source/class/qx/bom/webfonts/Validator.js

```diff
@@ -307,7 +306,7 @@ qx.Class.define("qx.bom.webfonts.Validator", {
       }
 
       var elem = document.createElement("span");
-      elem.innerHTML = comparisonString || qx.bom.webfonts.Validator.COMPARISON_STRING;
+      elem.innerHTML = this.getComparisonString();
```

Comment:
Ok. I'll revert the last commit. The initial version was correct, because we need to distinguish between two types of comparison strings.